src/lomc.jl

```diff
@@ -187,8 +174,7 @@ required for continuation runs.
 * `r_strat::ReportingStrategy = EveryTimeStep()` - see [`ReportingStrategy`](@ref)
 * `τ_strat::TimeStepStrategy = ConstantTimeStep()` - see [`TimeStepStrategy`](@ref)
 * `m_strat::MemoryStrategy = NoMemory()` - see [`MemoryStrategy`](@ref)
-* `num_replicas` - set the number of replica runs to run.
-* `operator` - set an operator to use with replicas.
+* `replica::ReplicaStrategy = NoReplica()` - see [`ReplicaStrategy`](@ref).
```

Comment:
```suggestion
* `replica::ReplicaStrategy = NoStats(1)` - see [`ReplicaStrategy`](@ref).
```
I'm not sure whether the "1" should be in there. Maybe it is useful to indicate that a single "replicum" will be propagated.